PWGHF/D2H/Tasks/taskDstarToD0Pi.cxx

```diff
@@ -179,8 +186,11 @@ struct HfTaskDstarToD0Pi {
       auto collision = candDstar.collision_as<CollisionsWCent>();
       auto centrality = collision.centFT0M(); // 0-100%
 
+      registry.fill(HIST("Efficiency/hPVContributorsEventsWCand"), collision.numContrib(), centrality);
+
       auto signDstar = candDstar.signSoftPi();
       if (signDstar > 0) {
+        registry.fill(HIST("Efficiency/hPVContributorsEventsWCandSignal"), collision.numContrib(), centrality, (invDstar - invD0));
```

Comment:
Same for this (you can check in the candidate loop if you have at least one and after the candidate loop you fill the histo)